src/node/BlockBuilder.cpp

```diff
@@ -43,7 +43,7 @@ bool BlockBuilder::AddTransaction(const Transaction::CPtr& pTransaction, const s
 
     // Validate transaction
     pTransaction->GetBody().Validate();
-    BlockSumValidator::ValidateForTx(*pTransaction);
+    KernelSumValidator::ValidateForTx(*pTransaction);
```

Comment:
Should this be pTransaction->Validate() ?

Reply:
Yes, great catch!